src/constants/constants.js

```diff
@@ -586,3 +588,16 @@ export const BLOCK_TIME_MILLIS = {
 };
 
 export const TRANSACTION_CONFIRMATIONS = 1;
+
+export const CACHED_FUNCTIONS = {
```

Comment:
Could you also please move the [cached values](https://github.com/OriginTrail/ot-node/blob/d8bffc33a42ae29c096798176ffe8381d88f19a3/src/service/service-agreement-service.js#L43) from the scoring function ?